src/core_test/mpas_test_core.F

```diff
@@ -167,6 +168,14 @@ function test_core_run(domain) result(iErr)!{{{
          call mpas_log_write('Stream I/O tests: FAILURE', MPAS_LOG_ERR)
       end if
 
+      iErr = 0
+      ! Test string split function
+      call mpas_test_split_string(iErr)
+      if ( iErr == 0 ) then
+          call mpas_log_write('Split Strings tests: SUCCESS')
```

Comment:
Admitting that the current output from most of the `test` core tests is barely readable, we could nonetheless make the logged message for the string tests stand out a bit more. Right now, we get something like this in the log file:
```
 Stream I/O tests: SUCCESS
 Split Strings tests: SUCCESS
  Performing time interval tests
```
What do you think about modifying the print statements here so that the result in the log file looks more like this:
```
 Stream I/O tests: SUCCESS
           
 String utils tests: 
   mpas_split_string: SUCCESS
   
  Performing time interval tests
```
There could be a blank line, a generic line indicating that string utils will be tested, and then a single line for each function that we test. This seems like it might be a bit more extensible as we add more tests for the string utilities in future.

Reply:
yep, I like that change.